Classes/Issues/GithubClient+Issues.swift

```
@@ -183,4 +184,16 @@ extension GithubClient {
         }))
     }
     
+    func deleteComment(owner: String, repo: String, commentID: Int, completion: @escaping (Result<Bool>) -> ()) {
+        request(Request(path: "repos/\(owner)/\(repo)/issues/comments/\(commentID)", method: .delete, completion: { (response, _) in
+            // As per documentation this endpoint returns no content, so all we can validate is that
+            // the status code is "204 No Content".
+            if response.response?.statusCode == 204 {
+                completion(.success(true))
+            } else {
+                completion(.error(nil))
```

I think we should propagate the error here.